bridge/opencensus/utils/utils.go

```diff
@@ -0,0 +1,57 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils // import "go.opentelemetry.io/otel/bridge/opencensus/utils"
+
+import (
+	"fmt"
+
+	octrace "go.opencensus.io/trace"
+
+	"go.opentelemetry.io/otel/global"
+	"go.opentelemetry.io/otel/trace"
+)
+
+// OTelSpanContextToOc converts from an OpenTelemetry SpanContext to an
+// OpenCensus SpanContext, and handles any incompatibilities with the global
+// error handler.
+func OTelSpanContextToOc(sc trace.SpanContext) octrace.SpanContext {
```

Comment:
Should the OpenCenesus initialism be capitalized?

```suggestion
func OTelSpanContextToOC(sc trace.SpanContext) octrace.SpanContext {
```